app/serializers/api/v1/mitigation/mitigation_action_serializer.rb

```diff
@@ -3,14 +3,17 @@
 # Table name: mitigation_actions
 #
 #  id                           :bigint(8)        not null, primary key
-#  mitigation_category_id       :integer
-#  name                         :string
+#  mitigation_theme_id          :integer
+#  name                         :text
 #  objectives                   :text
+#  mitigation_type              :string
 #  status                       :string
 #  actor                        :string
 #  time_horizon                 :string
 #  ghg                          :string
 #  estimated_emission_reduction :string
+#  cobenefits                   :string
+#  quantified_effect            :boolean
```

Comment:
There seem to be more changes to the annotation than the migration file suggests, is that right?

Reply:
I guess I just didn't run `annotate` on the previous commits. These fields have been changed in a previous PR, but didn't get annotated.